chrome-launcher.ts

```diff
@@ -118,15 +120,26 @@ export class Launcher {
     this.enableExtensions = defaults(this.opts.enableExtensions, false);
     this.connectionPollInterval = defaults(this.opts.connectionPollInterval, 500);
     this.maxConnectionRetries = defaults(this.opts.maxConnectionRetries, 50);
+
+    if (typeof this.opts.userDataDir === 'boolean') {
+      if (!this.opts.userDataDir) {
+        this.useDefaultProfile = true;
+      } else {
+        throw new Error('userDataDir must be false or a path');
+      }
+    } else {
+      this.useDefaultProfile = defaults(this.opts.forceDefaultProfile, false);
```

Comment:
do we need this now? I guess we could explicitly set `this.useDefaultProfile` to false now that forceDefaultProfile is gone